forms/TabSet.php

```diff
@@ -152,17 +152,18 @@ public function push(FormField $field) {
 	 * @param string $name Name of the field to insert before
 	 */
 	public function insertBefore($insertBefore, $field) {
-		parent::insertBefore($insertBefore, $field);
 		if($field instanceof Tab) $field->setTabSet($this);
 		$this->sequentialSet = null;
+		return parent::insertBefore($insertBefore, $field);
 	}
 
 	public function insertAfter($insertAfter, $field) {
-		parent::insertAfter($insertAfter, $field);
 		if($field instanceof Tab) $field->setTabSet($this);
 		$this->sequentialSet = null;
```

Comment:
This variable (sequentialSet) is a hangover from when CompositeField used to extend FieldList. It doesn't anymore, and is no longer useful.